index.html

```diff
@@ -5,47 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="./scss/index.css" />
-    <script src="./hamburger.js"></script>
   </head>
-  <body>
-    <header>
-      <div class="header_container">
-        <div class="logo">
-          <img src="./public/images/Dark=no.png" alt="logo" />
-        </div>
-        <nav>
-          <ul>
-            <li>
-              <a href="#">home</a>
-            </li>
-
-            <li>
-              <a href="#">register my dna kit</a>
-            </li>
-
-            <li>
-              <a href="#">features</a>
-            </li>
-
-            <li>
-              <a href="#">faq</a>
-            </li>
-
-            <li>
-              <a href="#">about</a>
-            </li>
-          </ul>
-          <!-- <div class="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-          </div> -->
-        </nav>
-        <button class="submit_button" type="button">order now</button>
```

Comment:
if you have some tools in header you can close them in another div called header_tools